vendor/github.com/openshift/api/config/v1/0000_10_config-operator_01_ingress.crd.yaml

```diff
@@ -81,6 +81,46 @@ spec:
                 domain:
                   description: "domain is used to generate a default host name for a route when the route's host name is empty. The generated host name will follow this pattern: \"<route-name>.<route-namespace>.<domain>\". \n It is also used as the default wildcard domain suffix for ingress. The default ingresscontroller domain will follow this pattern: \"*.<domain>\". \n Once set, changing domain is not currently supported."
                   type: string
+                loadbalancer:
```

Comment:
We're missing https://github.com/openshift/api/commit/24ee13c10f3bf7c3320392ba687a5a42bb3c04c4.

Reply:
Whoops, done